.github/workflows/build.yaml

```diff
@@ -254,13 +254,13 @@ jobs:
     permissions:
       contents: "read"
       id-token: "write"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Is there a reason this is latest instead of 24.04 LTS which the rest of the workflows use?

Reply:
the other jobs are building binaries; where dependent lib version etc. matter for the final build.

this job specifically, is only responsible for creating a checksum and uploading to cdn bucket. neither is dependent on ubuntu version or dependencies.

github runners are always LTS, latest just means 24.04 currently, 26.04 when it releases and so on.